inspector.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Flutter Widget Inspector
+
+permalink: /inspector/
+---
+
+The Flutter Widget Inspector is powerful tool for visualizing and exploring
+Flutter Widget trees.
+
+* TOC Placeholder
+{:toc}
+
+## Flutter Widget Inspector
```

Comment:
At the end of this page, maybe a link for where people ask for help and file bugs.

Reply:
done